rootfs/zpkg/zpkg

```diff
@@ -118,6 +118,12 @@ remove_package()
     echo Removing files:
     md5sum -c "$zdb"
 
+    # Remove pyc files that have been generated by Python and their __pycache__
+    # parent directories so that we can remove any empty directories in the 
+    # zdd file
+    find "$ROOT_DIR" -name '*.pyc' -delete
+    find "$ROOT_DIR" -name '__pycache__' -delete
+
```

Comment:
This is brute force, we remove *all* generated files any time we remove any package.
Unfortunately given the problem we're trying to solve and the crude tools available to us, I think this is the correct solution :disappointed: